src/index.html

```diff
@@ -1,12 +1,479 @@
+              <div class="list-stars">
+                <img 
+                  src="images/star-active.svg" 
+                  alt="star-feedback" 
+                  class="list-stars__star list-stars__star--active"
+                  >
+                <img 
+                  src="images/star-active.svg" 
+                  alt="star-feedback" 
+                  class="list-stars__star list-stars__star--active"
+                  >
+                <img 
+                  src="images/star-active.svg" 
+                  alt="star-feedback" 
+                  class="list-stars__star list-stars__star--active"
+                  >
+                <img 
+                  src="images/star-active.svg" 
+                  alt="star-feedback" 
+                  class="list-stars__star list-stars__star--active"
+                  >
+                <img 
+                  src="images/star.svg" 
+                  alt="star-feedback" 
+                  class="list-stars__star list-stars__star--inactive"
+                  >
+              </div>
+              <p class="feedback__count">Reviews: 5</p>
+            </div>
+            <p class="card__price">Price: <span class="card__price-digit card__price-digit--bold">$2,199</span></p>
+            <a href="#" >
+              <button class="card__button-buy card__button-buy--hover">BUY</button>
+            </a>
+          </div>
+        </div>
+      </section>
+
+      <section class="section">
+        <div class="card">
+          <div class="card__item">
+            <img 
+              src="images/imac.jpeg" 
+              alt="photo-imac" 
+              class="card__image">
+            <h3 class="card__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h3>
+            <p 
+              class="card__product-code"
+              >Product code: 195434
+            </p>
+            <div class="feedback">
+              <div class="list-stars">
+                <img 
+                  src="images/star-active.svg" 
+                  alt="star-feedback" 
+                  class="list-stars__star list-stars__star--active"
+                  >
+                <img 
+                  src="images/star-active.svg" 
+                  alt="star-feedback" 
+                  class="list-stars__star list-stars__star--active"
+                  >
+                <img 
+                  src="images/star-active.svg" 
+                  alt="star-feedback" 
+                  class="list-stars__star list-stars__star--active"
+                  >
+                <img 
+                  src="images/star-active.svg" 
+                  alt="star-feedback" 
+                  class="list-stars__star list-stars__star--active"
+                  >
+                <img 
+                  src="images/star.svg" 
+                  alt="star-feedback" 
+                  class="list-stars__star list-stars__star--inactive"
+                  >
```

Comment:
omg, that should not be there, stars should be set in styles, not in html

Reply:
what do you mean ? Set in style
only change src to backgroun or content?
Thanks!

Reply:
Set stars as a background-image